core-js/src/main/javascript/cdf-legacy/components/input.js

```diff
@@ -699,16 +699,23 @@ var DateRangeInputComponent = BaseComponent.extend({
     var inputSeparator = this.inputSeparator = this.inputSeparator || ">";
 
     if (this.singleInput == undefined || this.singleInput == true) {
-      dr = $('<input class="date-range-single-input" id="' + inputId + '" value="' + startValue + ' ' + inputSeparator + ' ' + endValue + '"/>');
+      dr = $('<input class="date-range-single-input" id="' + inputId + '"/>');
     } else {
-      dr = $('<input class="date-range-multiple-input" id="' + inputId + '" value="' + startValue + '"/>' + inputSeparator +
-             '<input class="date-range-multiple-input-2" id="' + inputId + '2" value="' + endValue + '"/>');
+      dr = $('<input class="date-range-multiple-input" id="' + inputId + '"/>' + inputSeparator +
+      '<input class="date-range-multiple-input-2" id="' + inputId + '2' + '"/>');
     }
 
     this.placeholder()
         .addClass('date-range-input-container')
         .html(dr);
 
+    if (this.singleInput == undefined || this.singleInput == true) {
+      $('#' + inputId).val(startValue + ' ' + inputSeparator + ' ' + endValue);
```

Comment:
Using selectors to search for elements that are being created here seems to be excessive and might affect performance.